config/production.yaml

```diff
@@ -101,8 +101,13 @@ API_PROXY_CORS_ALLOWED: http[s]?://.*nypl.org
 READER_VERSION: v2
 
 # ANALYTICS CONFIGURATION
+PUBLISHERS: "UofM"    # publishers need to be a comma separated string
+REFERRER_URL: "https://drb-qa.nypl.org/"
+
+# Downloads Reports
 DOWNLOAD_BUCKET: "ump-pdf-repository-logs"
 DOWNLOAD_LOG_PATH: "logs/946183545209/us-east-1/ump-pdf-repository/"
-REFERRER_URL: "https://drb-production.nypl.org/"
-# publishers need to be a comma separated string
-PUBLISHERS: "UofM"
+
+# Views Reports
+VIEW_BUCKET: "drb-files-qa-logs"
```

Comment:
`drb-files-production-logs`

Reply:
Just double checking, does this bucket exist yet? Not seeing it here: https://us-east-1.console.aws.amazon.com/s3/buckets?region=us-east-1&bucketType=general&region=us-east-1

Reply:
I don't think so. But let's switch!